app/View/Elements/alphabetSearch.ctp

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * COmanage Registry Alphabet Search Bar
+ *
+ * Portions licensed to the University Corporation for Advanced Internet
+ * Development, Inc. ("UCAID") under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * UCAID licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @link          http://www.internet2.edu/comanage COmanage Project
+ * @package       registry
+ * @since         COmanage Registry v4.0.0
+ * @license       Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ */
+
+// Globals
+global $cm_lang, $cm_texts;
+// Get a pointer to our model
+$model = $this->name;
+$controller = Inflector::tableize($model);
```

Comment:
tableize() adds an "s" to our model (in this case "co_peoples"). I suspect what we want is:
$controller = Inflector::underscore($model);

Reply:
Done